3languajesTemplate.js

```diff
@@ -0,0 +1,116 @@
+/*
+Write a html, css and javascript web page that:
+
+A submit form that takes the user:
+	name, lastname
+	phone number.
+	email address.
+	home address.
+	password.
+	secret question.
+Then a button that once clicked checks if all fields were full, if so
+It displays a json variable with all of the user information
+
+
+Chose a grayish background color and be sure to user a title sizing, and just play with css to make it look somewhat pretty.
+*/
+<form action="/action_page.php">
```

Comment:
you do not need php for this, since you have very few elements you could just give them unique ids